js/sign/src/integrityblock.ts

```diff
@@ -7,12 +7,34 @@ import {
   VERSION_B1,
 } from './constants.js';
 import { checkDeterministic } from './cbor/deterministic.js';
+import * as readline from 'readline';
+
+// A helper function that can be used to read the passphrase to decrypt a
+// password-decrypted private key.
+export async function readPassphrase(): Promise<string> {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+  return new Promise((resolve) =>
+    rl.question('Passphrase for the key: ', (passphrase) => {
+      // TODO(sonkkeli): Hide the password when being written.
```

Comment:
How about using https://www.npmjs.com/package/read as part of this PR to hide the password?